readme + get started link improvements (#108)
open-edge-platform/edge-microvisor-toolkit: docs/developer-guide/get-started.md

### Edge Microvisor Toolkit Standalone

> [Download the ISO installer](https://edgesoftwarecatalog.intel.com/package/licenseagreement/edge_microvisor_toolkit_standalone_node) for the standalone Edge Node from the Intel® Edge Software Catalog.

Author: The download link was changed to (unskippable) license agreement, because system and version selection don't do anything at the moment.

Reviewer: @magerstam, is the selector going to receive more options? Or maybe it's a bug in the page itself?

Reviewer 2: I don't think the suggested link will work. I think you have to land first on the edge_microvisor_toolkit_standalone_node page first, make your selection and proceed to download. Can you please revert. Going directly to the licenseagreement will just create a spinner that doesn't proceed...

Reviewer: Oh, so the license page needs parameters from the selector? If so, let's reword the text to indicate to readers what they are clicking. Maybe something like:
"Go to Intel® Edge Software Catalog and download a standalone Edge  Node ISO installer."

Reviewer 2: Yes, but let's preserve the link to the landing page for standalone ISO

Reviewer: yes, that's the idea, we preserve the link and adjust the wording to be clear that clicking it will redirect to a download page and not download the package (as opposed to "download ISO")

> [Download the ISO installer](https://edgesoftwarecatalog.intel.com/package/licenseagreement/edge_microvisor_toolkit_standalone_node) for the standalone Edge Node from the Intel® Edge Software Catalog.

Reviewer: ```suggestion
[Go to Intel® Edge Software Catalog](https://edgesoftwarecatalog.intel.com/package/edge_microvisor_toolkit_standalone_node) and download a standalone Edge Node ISO installer.
```
